routes/users.js

```diff
@@ -0,0 +1,142 @@
+const { mail } = require("./mail/mail");
+const index = require("../index");
+const { users: usersBd } = require("./sequelize/users");
+
+let users = {};
+
+users.signUp = async (req, res) => {
```

Comment:
usually there is separate routes for auth (`/auth`) and for fetching users.
Move all auth logic to separate route